aws/v13.1.0/release.diff

```diff
@@ -0,0 +1,70 @@
+# Generated with:                                                  # Generated with:
+# devctl release create --name 13.0.0 --base v12.7.0 --releases |  # devctl release create --name 13.1.0 --provider aws --base 13.
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v13.0.0                                                 |    name: v13.1.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.3.0                                              |      version: 1.6.1
+  - componentVersion: 1.0.2                                     |    - name: cert-manager
```

Comment:
why do we miss the `componentVersion` section in the new release?

Reply:
I added the missing componentVersion.